src/solvers/elasticity_solver.cpp

```diff
@@ -40,12 +40,7 @@ ElasticitySolver::ElasticitySolver(int order, std::shared_ptr<mfem::ParMesh> pme
 void ElasticitySolver::SetDisplacementBCs(std::vector<int> &                       disp_bdr,
                                           std::shared_ptr<mfem::VectorCoefficient> disp_bdr_coef, int component)
 {
-  SetEssentialBCs(disp_bdr, disp_bdr_coef, component);
-
-  // Get the list of essential DOFs
-  for (auto &ess_bc_data : m_ess_bdr) {
-    displacement.space->GetEssentialTrueDofs(ess_bc_data->bc_markers, ess_bc_data->true_dofs);
-  }
+  SetEssentialBCs(disp_bdr, disp_bdr_coef, *displacement.space, component);
```

Comment:
Just noticed displacement is a member variable that isnt prefixed with "m_" was confused where it was coming from because of this.

Reply:
Yep, I have this changed in my coupled solver branch.